databricks_cli/configure/config.py

```diff
@@ -37,11 +38,15 @@ def provide_api_client(function):
     """
     @six.wraps(function)
     def decorator(*args, **kwargs):
+        ctx = click.get_current_context()
+        command_name = "-".join(ctx.command_path.split(" ")[1:])
+        command_uuid = str(uuid.uuid1())
```

Comment:
Should UUID be generated per CLI commands? Here it seems to be generated for every instance of API client. Also what if the stack CLI command calls other CLI methods (will there be any duplicated records)?